src/main/java/org/trigon/hardware/rev/spark/io/RealSparkIO.java

```diff
@@ -45,76 +44,27 @@ public void stopMotor() {
     }
 
     @Override
-    public void setPeriodicFramePeriod(CANSparkLowLevel.PeriodicFrame frame, int periodMs) {
-        motor.setPeriodicFramePeriod(frame, periodMs);
+    public void setPeriodicFrameTimeout(int periodMs) {
+        motor.setPeriodicFrameTimeout(periodMs);
     }
 
     @Override
-    public void setReference(double value, CANSparkBase.ControlType ctrl, int pidSlot, double arbFeedForward) {
+    public void setReference(double value, SparkBase.ControlType ctrl, int pidSlot, double arbFeedForward) {
         pidController.setReference(value, ctrl, pidSlot, arbFeedForward);
     }
 
     @Override
-    public void setReference(double value, CANSparkBase.ControlType ctrl, int pidSlot, double arbFeedForward, SparkPIDController.ArbFFUnits arbFFUnits) {
+    public void setReference(double value, SparkBase.ControlType ctrl, int pidSlot, double arbFeedForward, SparkClosedLoopController.ArbFFUnits arbFFUnits) {
         pidController.setReference(value, ctrl, pidSlot, arbFeedForward, arbFFUnits);
     }
 
     @Override
-    public void setBrake(boolean brake) {
-        motor.setIdleMode(brake ? CANSparkMax.IdleMode.kBrake : CANSparkMax.IdleMode.kCoast);
+    public void configure(SparkMaxConfig configuration, SparkBase.ResetMode resetMode, SparkBase.PersistMode persistMode) {
+        motor.configure(configuration, resetMode, persistMode);
     }
 
     @Override
     public void setInverted(boolean inverted) {
         motor.setInverted(inverted);
     }
-
-    @Override
-    public void enableVoltageCompensation(double voltage) {
-        motor.enableVoltageCompensation(voltage);
-    }
-
-    @Override
-    public void setClosedLoopRampRate(double rampRate) {
-        motor.setClosedLoopRampRate(rampRate);
-    }
-
-    @Override
-    public void setSmartCurrentLimit(int limit) {
-        motor.setSmartCurrentLimit(limit);
-    }
-
-    @Override
-    public void setOpenLoopRampRate(double rampRate) {
-        motor.setOpenLoopRampRate(rampRate);
-    }
-
```

Comment:
Why is all this stuff gone?

Reply:
REV changed how the configuration works. Instead of having lots of setters, they now do it like CTRE